### PR TITLE
Add Konstantina's slides next to Francesco's

### PR DESCRIPTION
**Pull request by ntinaxenou**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -100,7 +100,180 @@
             <br>
             <p>Thanks!</p>
         </section>
-    </div>
+		<section>
```
Comment (FrancescoBorzi) [inline index.html:103]: may you please align the indentation?

File: css/theme/black.css [hunk 2]
```diff
@@ -176,6 +176,8 @@ body {
 .reveal table th,
 .reveal table td {
   text-align: left;
+  color: white;
+  font-size: 0.6em;
```
Comment (FrancescoBorzi) [inline css/theme/black.css:180]: do we really need those changes? If so, I would suggest to make them in a separate css file, without touching the css theme file of reveal.js

**Comment by FrancescoBorzi** [comment]
also a few notes:

- maybe it would be better to move the images under your own directory? something like ``/img/konstantina``
- can you please remove the ``.DS_Store`` file?

File: css/theme/black.css (hunk 2 above)
Comment (ntinaxenou) [reply css/theme/black.css:180]: You are right, thanks!

File: index.html (hunk 1 above)
Comment (ntinaxenou) [reply index.html:103]: Sure!

**Pull request merged**